tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+skipsdist = True
+envlist = upgrade
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}:{toxinidir}/src
+    PYTHONBREAKPOINT=ipdb.set_trace
```

Comment:
This seems like it will interfere with CI if a traceback occurs. Will the process just hang?